value.go

```diff
@@ -837,6 +842,7 @@ func (vlog *valueLog) write(reqs []*request) error {
 		return nil
 	}
 
+	valueSizes := make(ReqValSize, 0)
```

Comment:
This needs to be inside the for loop. Also, we can specify the capacity upfront. 
`make(ReqValSiz, 0, len(req.Entries))`